src/lib/Scenes/Fair/Screens/__tests__/FairArtists-tests.tsx

```diff
@@ -20,7 +19,7 @@ describe("FairArtists", () => {
       `,
       mockData: {
         fair: fairFixture,
-      } as FairArtistsTestsQueryRawResponse,
+      },
```

Comment:
In cases like these the idea was to actually fix the test, but that can be followed up on. I’m adding these typings back but comment them out for others to easily fix these disabled tests, if needed. /cc @ashfurrow

Reply:
(There’s a bunch more disabled tests related to fixture data not being correct. I’m not going to list them all out, you can find them as being disabled.)

Reply:
Ah, I didn't see the `xit` on the test; i was running `yarn jest` and then fixing the returned failures.